robot_description/srdf/demo.srdf

```diff
@@ -0,0 +1,563 @@
+<robot name="demo">
```

Comment:
The robot_description package needs either to go into the resources folder of in another package.
But I would put it in the resources folder at first.